### PR TITLE
polished Ember Inspector installation to match style guide

### PR DESCRIPTION
**Pull request by mattw59**
First PR to address Issue #602

### CODE REVIEW

File: source/ember-inspector/installation.md [hunk 1]
```diff
@@ -1,22 +1,19 @@
-The Ember Inspector extends browsers' developer tools to make understanding and debugging your Ember.js applications easier.
+The Ember Inspector is a browser add-on designed to help you understand and debug your Ember applications.
 
-Steps to install the inspector for Google Chrome, Firefox, other
-browsers (Bookmarklet), and apps running on mobile devices are explained
-below.
+You can install the Inspector on Google Chrome, Firefox, other
+browsers (via a bookmarklet), and on mobile devices. Simply follow the steps below.
```
Comment (locks) [inline source/ember-inspector/installation.md]: `and on mobile devices by following the steps below`

File: source/ember-inspector/installation.md [hunk 2]
```diff
@@ -64,38 +57,34 @@ and click on the `Ember` tab.
...
 
-To open the inspector, just click on the bookmark.
-
-Internet Explorer will open an iframe instead of a popup due to the lack of support for cross-origin messaging.
-
-Safari blocks popups by default, so you'll need to enable popups before
-using this option.
+To open the Inspector, just click on the new bookmark.
 
+Because it lacks support for cross-origin messaging, Internet Explorer will open an iFrame, not a popup. Safari blocks popups by default, so you'll need to enable popups before using the bookmarklet.
```
Comment (locks) [inline source/ember-inspector/installation.md]: I don't think the IE information is useful here.

**Comment by locks** [comment]
Other than the two comments this looks good :)

**Comment by mattw59** [comment]
Update to address @locks's comments.

File: source/ember-inspector/installation.md [hunk 3]
```diff
@@ -1,22 +1,19 @@
...
+Tool. To begin, visit the Extension page on the [Chrome Web Store][ember-inspector-chrome].
 
 Click on "Add To Chrome":
 
 <img src="../../images/guides/ember-inspector/installation-chrome-store.png" width="680" />
 
-Once installed, go to an Ember.js application, open the developer tools,
-and click on the `Ember` tab next to the rest of the devtools panels.
+Once installed, go to an Ember application, open the Developer Tools,
+and click on the `Ember` tab at the far right.
```
Comment (locks) [inline source/ember-inspector/installation.md]: might not be the right-most tab, but I think that's being too precious. :+1:

**Pull request merged**